priv/templates/phoenix.gen.json/changeset_view.ex

```diff
@@ -1,9 +1,18 @@
 defmodule <%= base %>.ChangesetView do
   use <%= base %>.Web, :view
 
+  @doc """
+  Traverses and translates changeset errors.
+
+  See `Ecto.Changeset.traverse_errors/2` for more details.
+  """
+  def translate_errors(changeset) do
+    Ecto.Changeset.traverse_errors(changeset, &translate_error(&1))
```

Comment:
bikeshed, but I always prefer &1 because it's less munging text around when arity later changes. I could go either way though, so will change if you are strong on `&translate_error/1`